src/components/Sum/Sum.jsx

```diff
@@ -1 +1,10 @@
-// export const Sum = () => ();
+export const Sum = ({ a, b }) => (
+  <p>
+    { `Sum of ${a} and ${b} is ${a + b}` }
+  </p>
+);
+
+Sum.defaultProps = {
+  a: 0,
+  b: 0,
+};
```

Comment:
The defaultProps is deprecated. You can assign them with default parameters.
`({ a = 0, b = 0 }) =>`